src/Concerns/PerformsCharges.php

```diff
@@ -89,6 +89,10 @@ public function createPayment($amount, array $options = [])
             $options['customer'] = $this->stripe_id;
         }
 
+        if ($options['confirm'] ?? false) {
+            $options['return_url'] ??= route('home');
+        }
+
```

Comment:
When a payment intent is being confirmed, we'll always need to supply a return url.